src/store/actions.ts

```diff
@@ -886,8 +886,7 @@ export const startQuickSearch = async ({ commit, getters }) => {
       .replace(/¢/g, 'C')
       .replace(/(`|~|!|\||\(|\)|\[|\]|\{|\}|:|"|\^|#|%|\?|,)/g, '')
 
-    // TODO: Refactor into metho that calls an action
-    await this.getQuickSearch({ 'exactMatch': exactMatchName, 'synonymMatch': synonymsName })
+    await getQuickSearch({ commit, getters }, { 'exactMatch': exactMatchName, 'synonymMatch': synonymsName })
```

Comment:
Are you sure we should keep methods that make network calls in the store? I'm a bit uncomfortable with that as it might be part of the async issues we've seen in Prod. I'd be more comfortable if store actions didn't call async things.

Reply:
I'm with you 100%.
I do have an urgency to get the app back to a functional state before refactoring too too much though. 

If you like, since we are still waiting for a discussion with Thor/Kial on what to do with nr creation/payment, my next commit can focus on this exact type of refactoring?

Reply:
I'm content with quick merges at the moment, even if the code is partly broken or commented out or mid-refactor. As long as we're never far off a working codebase.

I guess the old code had these network calls in the store already, so ^^ isn't any worse.

Reply:
Already working on it!